crates/std_detect/src/detect/os/windows/aarch64.rs

```diff
@@ -0,0 +1,30 @@
+//! Run-time feature detection for Aarch64 on Windows.
+
+use crate::detect::{cache, Feature};
+use winapi::um::processthreadsapi::IsProcessorFeaturePresent;
+use winapi::um::winnt::*;
+
+/// Try to read the features from OS API.
+pub(crate) fn detect_features() -> cache::Initializer {
+    let mut value = cache::Initializer::default();
+    {
+        let mut enable_feature = |f, enable| {
+            if enable {
+                value.set(f as u32);
+            }
+        };
+        enable_feature(Feature::asimd, unsafe {
+            IsProcessorFeaturePresent(PF_ARM_NEON_INSTRUCTIONS_AVAILABLE) != 0
+        });
+        enable_feature(Feature::crc, unsafe {
+            IsProcessorFeaturePresent(PF_ARM_V8_CRC32_INSTRUCTIONS_AVAILABLE) != 0
+        });
+        enable_feature(Feature::crypto, unsafe {
+            IsProcessorFeaturePresent(PF_ARM_V8_CRYPTO_INSTRUCTIONS_AVAILABLE) != 0
+        });
+        enable_feature(Feature::pmull, unsafe {
+            IsProcessorFeaturePresent(PF_ARM_V8_CRYPTO_INSTRUCTIONS_AVAILABLE) != 0
```

Comment:
copy paste error?

Reply:
On Windows/aarch64, `PF_ARM_V8_CRYPTO_INSTRUCTIONS_AVAILABLE` means that aes, sha1, sha2 and pmull support.

Reply:
Maybe add a comment?